jobs/base_model_validator.py

```diff
@@ -66,6 +67,29 @@ def process(self, input_model, regex_string):
             yield errors.ModelInvalidIdError(model)
 
 
+class ValidatePostCommitIsPrivate(beam.DoFn):
+    """DoFn to check if post_commmit_status is private when
+    post_commit_is_private is true and vice-versa."""
+
+    def process(self, input_model):
+        """Function validates that post_commit_is_private is true iff
+        post_commit_status is private
+
+        Args:
+            input_model: datastore_services.Model. Entity to validate.
```

Comment:
This is only applicable to `BaseCommitLogEntryModel`, right? Please clarify in the doc string if so.

Reply:
Done